### PR TITLE
ref/update_example

### PR DESCRIPTION
**Pull request by alhiwatan**
Update the example to support the dark mode.  Also, update the UI to mimic the RN.

Screenshot 2024-09-11 at 5 34 46 PM

Screenshot 2024-09-11 at 5 35 44 PM

### CODE REVIEW

File: applovin_max/example/pubspec.yaml [hunk 1]
```diff
@@ -30,3 +30,5 @@ dev_dependencies:
 
 flutter:
   uses-material-design: true
+  assets:
+    - assets/applovin_logo.png
```
Comment (thomasmso) [inline applovin_max/example/pubspec.yaml:34]: Curious, but =what would be the consequences of omitting this?

**Review by thomasmso — APPROVED** [review]
Do we need to delete the old logo?

File: applovin_max/example/pubspec.yaml (hunk 1 above)
Comment (alhiwatan) [reply applovin_max/example/pubspec.yaml]: Omitting results in an exception: 'Asset not found.'  But, I realized this needs only the directory. 
I tested it without `/`.

```
  assets:
    - assets/
```

**Comment by alhiwatan** [comment]
Scale the image, and delete the old icons.

Screenshot 2024-09-12 at 10 54 49 AM

**Review by thomasmso — APPROVED** [review]

**Pull request merged**